app/src/main/res/values/strings.xml

```diff
@@ -188,9 +130,9 @@
     <string name="state_obtaining_ip">Obtaining IP</string>
     <string name="result_state">Result</string>
     <string name="state_connected">Connected</string>
-    <string name="state_connection_failed">Connection Failed</string>
+    <string name="state_connection_failed">Connection failed</string>
 
-    <string name="version">ProvVer: %d</string>
```

Comment:
Ver? V? It's in the UI element where shorter is better.

Reply:
Ver or V is ok. I just find ProvVer weird.